### DISCUSSION TITLE
Communication with Pumpe directly

### DISCUSSION THREAD

**Discussion by michaelschaaf** [Yogui79/IntexPureSpa#45] [General]
Hi, I'm really impressed about this solution to make my PureSpa smart. I'm running a 28462 model and everything works like a charm when the remote control is connected to the pumpe. Without the remote control no commands are executed. But I would like to keep the remote control inside my house and want to communicate with the pumpe directly for daily Sanitation. Is this possible, and I only need to reconfigure my system, or needs the remote unit to be connected to the pumpe all the time?

**Comment by LEECHER1** [comment]
Sorry for my English.

As far as we know, it doesn't work.  The controller must always have a connection and must be at least 5m away from the pump.  ESP must be 5m away and also have WIFi.  You can use a WiFi extender and leave the ESP near the pump.

you can change the transmission power in code ESP/LC12s but it is set to the optimal level.  Section //RF Power
